src/commands/config.ts

```diff
@@ -67,7 +67,7 @@ export const configValidators = {
   [CONFIG_KEYS.OPENAI_MAX_TOKENS](value: any) {
     validateConfig(
       CONFIG_KEYS.OPENAI_MAX_TOKENS,
-      typeof value === 'number',
+      isNaN(+value),
```

Comment:
it should be `typeof value === 'number' && !isNaN(+value)`

Reply:
Tnx! Can you please take a look?

Reply:
this was just addressed in #176, thanks

could you close the PR please if you are happy with #176